src/crossfilter.js

```diff
@@ -121,6 +138,11 @@ function crossfilter() {
     offset = tmp.offset;
     one = tmp.one;
     zero = ~one;
+      
+    // Expose the dimension mask for internal use
+    dimension._offset = offset;
```

Comment:
Don't think you need to set anything, I think you just need to add an `id()` method returning `offset` - effectively the closure is the dimension object here, so `offset` sticks around as long as the dimension does.